src/screens/About/Team/ActiveMember.tsx

```diff
@@ -8,21 +8,17 @@ const ActiveMember: React.FC<{
   teamMember: UserProfile;
   includeTitle: boolean;
 }> = ({ teamMember, includeTitle }) => {
+  // ensure no trailing spaces from CMS
+  const fullName = teamMember.fullName?.trim();
+  const description = teamMember.description?.trim();
```

Comment:
nit: This could be optionally done in the CMS module where we load team data (see [src/cms-content/team/index.ts#L35](https://github.com/covid-projections/covid-projections/blob/a5b8e3488bce3887c08edc2a8bb12a012753afe8/src/cms-content/team/index.ts#L35-L35))

Reply:
nice idea, thanks : )